XamarinCommunityToolkitSample/Resx/AppResources.Designer.cs

```diff
@@ -10,35 +10,48 @@
 
```

Comment:
@jfversluis can we add this file to "git ignore"?

That's always regenerated with hundreds of lines



Can we discard this file?

Reply:
Yeah I've looked into that. Not sure if that will work OK every time with regenerating on the developer side, but I guess it's worth a try